### DISCUSSION TITLE
Mobile PDF

### DISCUSSION THREAD

**Discussion by fehart20** [pretix/pretix#3696] [Q&A] [answered]
Hi pretix-Team,

just a quick question: Where can I find the "Mobile PDF"-plugin? I searched in the marketplace but only found "Passbook Tickets".
Or is this only available for Enterprise / SaaS?

Thanks for your reply!
Felix

**Comment by c0de-bender** [comment]
I was also occasionally looking for plugins, which are present in the pretix.eu hosted service, but which I couldn't find on the marketplace or elsewhere. Is there a list of all those plugins?

**Reply by raphaelm** [reply]
Those are most likely new ones that have not yet been added to the marketplace for a lack of time. Once in a while, I try to add all forgotton ones, but they keep growing so quickly ;)

**Comment by raphaelm** [comment, marked as answer]
It's currently only available for Enterprise / SaaS
